_pages/relays/s0str.md

```diff
@@ -0,0 +1,12 @@
+---
+title: s0str
+permalink: /s0str/
+web: 
+github: metasikander/s0str
+tags:
+- relay
+programming language: Rust
```

Comment:
You convinced me that progLang should be an array as we might have cases with more than one language:

```suggestion
progLang:
- Rust
```

Reply:
[👍](https://emojipedia.org/thumbs-up/)